.gitignore

```diff
@@ -4,3 +4,11 @@ __pycache__/
 /dist/
 
 /.install.*.bash.inc
+
+.envrc
```

Comment:
for the moment I'm under the assumption that people have configured their aws credentials under profiles in `~/.aws/credentials`, such that aws cli commands can authenticate against a profile name.

because beanstalk is old, rather than `AWS_PROFILE`, it looks for `AWS_EB_PROFILE`; and, the `install` script checks that you have this set in your shell environment. (rather than worry about this, `manage` commands let your environment export down to `aws`/`eb`, which will complain if it doesn't find what it needs.)

personally, I use `direnv`, which loads `.envrc` files in the current directory. however, it doesn't seem as though this project currently has any need of enforcing this convention or tooling.